### PR TITLE
[ChopBox] [##102129484] Help Center

### PR DESCRIPTION
**Pull request by ooduye**
These are the help center and privacy pages for the ChopBox project

### CODE REVIEW

**Comment by andela-potemuyiwa** [comment]
Why do you have head and header file?

**Comment by ooduye** [comment]
I have made all the corrections. Please review again and merge.

File: app/Http/Controllers/HomeController.php [hunk 1]
```diff
@@ -37,9 +37,9 @@ public function index()
 	{   
 		$user = Auth::user();
 		if ($user->profile_state) {
-			return view('home');
+			return view('pages.home');
 		}else {
-			return view('first_profile');
+			return view('pages.first_profile');
```
Comment (unicodeveloper) [inline app/Http/Controllers/HomeController.php:42]: Change the name of the blade file to `initial_profile_update`

File: app/Http/Controllers/WelcomeController.php [hunk 2]
```diff
@@ -34,12 +34,12 @@ public function index() {
 		if (Auth::check ()) {
 			$user = Auth::user ();
 			if ($user->profile_state) {
-				return view ( 'home' );
+				return view ( 'pages.home' );
 			} else {
-				return view ( 'first_profile' );
+				return view ( 'pages.first_profile' );
```
Comment (unicodeveloper) [inline app/Http/Controllers/WelcomeController.php]: Change the name of the blade file to initial_profile_update

**Comment by ooduye** [comment]
I have changed the file name from first_profile to initial_profile_update. Please review again and merge.

**Pull request closed without merging**